frontend/src/app/app-events.service.ts

```diff
@@ -49,12 +50,80 @@ export class AppEventsService implements OnDestroy {
 
     // TODO
     let sumPosesDetected = 0;
+    interface Frame {
+      rwx: number;
+      rwy: number;
+      rsx: number;
+      rsy: number;
+      lwx: number;
+      lwy: number;
+      lsx: number;
+      lsy: number;
```

Comment:
Nit: Refactor properties to use readable, less cryptic names (e.g. `rightWristX`)